spark/v4.0/spark/src/main/java/org/apache/iceberg/spark/SparkUtil.java

```diff
@@ -154,8 +154,9 @@ public static Configuration hadoopConfCatalogOverrides(SparkSession spark, Strin
     // Find keys for the catalog intended to be hadoop configurations
     final String hadoopConfCatalogPrefix = hadoopConfPrefixForCatalog(catalogName);
     final Configuration conf = spark.sessionState().newHadoopConf();
+
     spark
-        .sqlContext()
```

Comment:
The generic [abstract class SparkSession](https://github.com/apache/spark/blob/master/sql/api/src/main/scala/org/apache/spark/sql/SparkSession.scala#L63) doesn't have this API `sqlContext` any more